trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/upload_reel/UploadReelViewModel.kt

```diff
@@ -76,7 +81,8 @@ internal class UploadReelViewModel(
     }
 
     private fun onValidationError(errorState: ErrorState) {
-        updateState { copy(errorState = errorState) }
+        val uploadReelError = mapToUploadReelError(errorState)
+        updateState { copy(errorState = uploadReelError) }
```

Comment:
what's your opinion to add this line in update direct -> mapToUploadReelError(errorState) 
I think not need anew variable or function Like this :
```suggestion
        updateState { copy(errorState = mapToUploadReelError(errorState)) }
```